### DISCUSSION TITLE
CSV Wide date format ISO 8601 in the validator

### DISCUSSION THREAD

**Discussion by sklukasatphs** [CMSgov/hospital-price-transparency#105] [Q&A] [answered]
As other users have mentioned the date format YYYY-MM-DD when the .csv opens out of Excel the date automatically goes to MM-DD-YYYY. we tried adding apostrophes to keep the format, ran it through the validator and received the following error:

"last_updated_on" value "2024-07-01" is not a valid format. You must encode the date using the ISO 8601 format: YYYY-MM-DD or the month/day/year format: MM/DD/YYYY, M/D/YYYY

is there an update to the regulation that approves of either MM/DD/YYYY, M/D/YYYY formats? Based on the validator results we cannot assume that a non ISO 8601 format is acceptable.

**Comment by tthirtythree** [comment]
We also have this question/issue. Some CMS Guidance would be helpful as the date draws near.

**Comment by RidenourK** [comment, marked as answer]
@sklukasatphs and @tthirtythree Thank you for your questions. 

Please see the new “[additional last updated on](https://github.com/CMSgov/hospital-price-transparency/tree/master/documentation/CSV#additional-notes-on-last_updated_on)” notes in the “[CSV documentation](https://github.com/CMSgov/hospital-price-transparency/tree/master/documentation/CSV)” section of this GitHub: ISO 8601 is the required format but the M/D/YYYY (e.g., 7/1/2024) or MM/DD/YYYY (e.g., 07/01/2024) formats are also accepted for the “last_updated_on” data element in CSV "Tall" or "Wide" MRFs.
